src/User.java

```diff
@@ -0,0 +1,9 @@
+public class User extends Guest{
```

Comment:
why user extends guest?

Reply:
guest IS A user

Reply:
So, can i rename it as an authorizedUser? Because any1 Who visits the Shop is already guest and there is no point in creating parent class to it as for me.

Reply:
I meant that no need to extend from user. Make user abstract and derive all kind of users you want